### PR TITLE
refactor: Adjust namespaces related to latest Uno packages

### PR DESCRIPTION
**Pull request by agneszitte-nventive**
## PR Type

What kind of change does this PR introduce?

- Refactoring (no functional changes, no api changes)

## What is the new behavior?

Adjust namespaces related to latest Uno packages

## PR Checklist

Please check if your PR fulfills the following requirements:

- [X] Tested code with current [supported SDKs](../README.md#supported)
- [X] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](doc/articles/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests](doc/articles/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [X] Contains **NO** breaking changes
- [ ] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
- [ ] Associated with an issue (GitHub or internal)

### CODE REVIEW

**Comment by agneszitte-nventive** [comment]
**PR Ready to review, **BUT DO NOT MERGE****. Tests are still in progress for some platforms

**Review by francoistanguay — APPROVED** [review]

File: src/Uno.Extensions.Templates/content/unoapp-extensions/MyExtensionsApp.Shared/Views/ProductsPage.xaml [hunk 1]
```diff
@@ -5,10 +5,10 @@
 	  xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
 	  xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
 	  xmlns:services="using:MyExtensionsApp.Services"
-	  xmlns:nav="using:Uno.Extensions.Navigation.UI"
-	  xmlns:reactive="using:Uno.Extensions.Reactive.UI"
+	  xmlns:uen="using:Uno.Extensions.Navigation.UI"
+	  xmlns:uer="using:Uno.Extensions.Reactive.UI"
```
Comment (Xiaoy312) [inline src/Uno.Extensions.Templates/content/unoapp-extensions/MyExtensionsApp.Shared/Views/ProductsPage.xaml:9]: duplicated xmlns (twice); please check your changes with "xamlns viewer.linq"

File: src/extensions/ExtensionsSampleApp/ExtensionsSampleApp.Shared/Views/Twitter/TwitterPage.xaml [hunk 2]
```diff
@@ -1,44 +1,43 @@
...
+      xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
+      xmlns:uen="using:Uno.Extensions.Navigation.UI"
+      mc:Ignorable="d"
       Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
 
     <Grid>
-        <controls:TabView nav:Region.Attached="true"
-                          nav:Region.Name="TwitterPage"
+        <muxc:TabView uen:Region.Attached="true"
+                          uen:Region.Name="TwitterPage"
```
Comment (Xiaoy312) [inline src/extensions/ExtensionsSampleApp/ExtensionsSampleApp.Shared/Views/Twitter/TwitterPage.xaml:14]: alignment

File: src/Uno.Extensions.Templates/content/unoapp-extensions/MyExtensionsApp.Shared/Views/ProductDetailsPage.xaml [hunk 3]
```diff
@@ -115,7 +114,7 @@
 										   x:Uid="_Reviews"
 										   Text="Reviews"
 										   Style="{StaticResource MaterialHeadline6}" />
-								<controls:RatingControl MaxRating="5"
+								<muxc:RatingControl MaxRating="5"
 														Value="{Binding Rating, Mode=TwoWay}" />
```
Comment (Xiaoy312) [inline src/Uno.Extensions.Templates/content/unoapp-extensions/MyExtensionsApp.Shared/Views/ProductDetailsPage.xaml:118]: alignment

File: src/extensions/ExtensionsSampleApp/ExtensionsSampleApp.Shared/UnoCommerce/CommerceHomePage.xaml [hunk 4]
```diff
@@ -43,22 +44,22 @@
             </VisualStateGroup>
         </VisualStateManager.VisualStateGroups>
         <muxc:NavigationView x:Name="NavView"
-                             nav:Region.Attached="true"
+                             uen:Region.Attached="true"
                              >
```
Comment (Xiaoy312) [inline src/extensions/ExtensionsSampleApp/ExtensionsSampleApp.Shared/UnoCommerce/CommerceHomePage.xaml:48]: odd formatting

File: src/extensions/ExtensionsSampleApp/ExtensionsSampleApp.Shared/UnoCommerce/ProductsPage.xaml [hunk 5]
```diff
@@ -37,12 +37,12 @@
                             HorizontalAlignment="Right">
                 <TextBlock Text="{Binding FilterQuery}" />
                 <HyperlinkButton Content="Filter"
-                                 nav:Navigation.Request="!FilterPopup"
-nav:Navigation.Data="{Binding FilterQuery, Mode=TwoWay}"
+                                 uen:Navigation.Request="!FilterPopup"
+uen:Navigation.Data="{Binding FilterQuery, Mode=TwoWay}"
```
Comment (Xiaoy312) [inline src/extensions/ExtensionsSampleApp/ExtensionsSampleApp.Shared/UnoCommerce/ProductsPage.xaml:41]: alignment

File: src/extensions/ExtensionsSampleApp/ExtensionsSampleApp.Shared/Views/NavigationViewPage.xaml [hunk 6]
```diff
@@ -17,15 +16,15 @@
                 <muxc:NavigationViewItem Tag="home"
                                          Icon="Home"
                                          Content="Home"
-nav:Navigation.Request="./Content1"
+uen:Navigation.Request="./Content1"
```
Comment (Xiaoy312) [inline src/extensions/ExtensionsSampleApp/ExtensionsSampleApp.Shared/Views/NavigationViewPage.xaml:19]: alignment + `/>` placement (x2: L20 & L24)

**Review by Xiaoy312 — CHANGES_REQUESTED** [review]
mostly just fix: src\Uno.Extensions.Templates\content\unoapp-extensions\MyExtensionsApp.Shared\Views\ProductsPage.xaml
nice to have: there are few more formatting issues i didnt want to mention anymore
if you are doing global changes, at least give some attention to the 1-2 lines before and after

File: src/Uno.Extensions.Templates/content/unoapp-extensions/MyExtensionsApp.Shared/Views/ProductsPage.xaml (hunk 1 above)
Comment (agneszitte-nventive) [reply src/Uno.Extensions.Templates/content/unoapp-extensions/MyExtensionsApp.Shared/Views/ProductsPage.xaml:9]: done thanks

**Comment by agneszitte-nventive** [comment]
> mostly just fix: src\Uno.Extensions.Templates\content\unoapp-extensions\MyExtensionsApp.Shared\Views\ProductsPage.xaml nice to have: there are few more formatting issues i didnt want to mention anymore if you are doing global changes, at least give some attention to the 1-2 lines before and after

changes done, thank you!

**Review by Xiaoy312 — APPROVED** [review]

**Pull request merged**